src/current/_includes/releases/v24.1/v24.1.6.md

```diff
@@ -0,0 +1,87 @@
+- Added timers around key parts of the [changefeed]({% link v24.1/change-data-capture-overview.md %}) pipeline to help debug feeds experiencing issues. The `changefeed.stage.{stage}.latency` metrics now emit latency histograms for each stage. The metric respects the changefeed `scope` label for debugging specific feeds. [#131429][#131429]
+
+<h3 id="v24-1-6-operational-changes">Operational changes</h3>
+
+- Added a `ranges.decommissioning`, representing the number of ranges that have a replica on a [decommissioning node]({% link v24.1/node-shutdown.md %}?filters=decommission). [#130248][#130248]
+- You can now configure the log format for the [`stderr` log sink]({% link v24.1/configure-logs.md %}#output-to-stderr) by setting the `stderr.format` field in the [YAML configuration]({% link v24.1/configure-logs.md %}#yaml-payload). [#131534][#131534]
+
+<h3 id="v24-1-6-db-console-changes">DB Console changes</h3>
+
+- The [DB Console]({% link v24.1/ui-overview.md %}) will now show a notification alerting customers without an [Enterprise license]({% link v24.1/enterprise-licensing.md %}) to [upcoming license changes](https://www.cockroachlabs.com/enterprise-license-update/) with a link to more information. [#130426][#130426]
+
+<h3 id="v24-1-6-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in which `SHOW CLUSTER SETTING FOR VIRTUAL CLUSTER` would erroneously return `NULL` for some settings. [#128783][#128783]
+- Addressed a bug in the [upgrade]({% link v24.1/upgrade-cockroach-version.md %}) pre-condition for repairing descriptor corruption, which could lead to upgrade finalization being stuck. [#130518][#130518]
+- Fixed a potential memory leak in [changefeeds]({% link v24.1/change-data-capture-overview.md %}) using a [cloud storage sink]({% link v24.1/changefeed-sinks.md %}#cloud-storage-sink). The memory leak could occur if both [`changefeed.fast_gzip.enabled`]({% link v24.1/cluster-settings.md %}#setting-changefeed-fast-gzip-enabled) and `changefeed.cloudstorage.async_flush.enabled` were `true`, and the changefeed received an error while attempting to write to the cloud storage sink. [#130601][#130601]
+- Fixed a bug in which some [`SELECT FOR UPDATE`]({% link v24.1/select-for-update.md %}) or [`SELECT FOR SHARE`]({% link v24.1/select-for-update.md %}) queries using `NOWAIT` could still block on locked rows when using the `optimizer_use_lock_op_for_serializable` [session setting]({% link v24.3/session-variables.md %}) under [`SERIALIZABLE`]({% link v24.1/demo-serializable.md %}) isolation. This bug was introduced with [`optimizer_use_lock_op_for_serializable`]({% link v24.1/session-variables.md %}#optimizer-use-lock-op-for-serializable) in v23.2.0.
```

Comment:
After seeing [@mikeCRL's comment](https://github.com/cockroachdb/docs/pull/19012#discussion_r1804863594), I checked if all the rn's had PR links. I saw that this release note is also missing its PR link. It should have PR #130429.

Sorry for missing this check during my review.